README.md

```diff
@@ -86,6 +86,16 @@ If you don't want to use multi-room or you only have one device, you can disable
 
 **Note:** Multi-room requires a network router that supports IP multicast/broadcast (most modern routers do).
 
+### Client Only Multi-room
+
+By default, balenaSound let's you connect to every device available. If you prefer to have one or multiple devices "hidden" in the network but still capable to play the multi-room audio you can enable the Client Only mode.
```

Comment:
Looks good! A few suggestions to better convey the message:

`By default, balenaSound let's you connect and stream audio from every device on the fleet. If you prefer to have one or more devices "hidden" in the network but still capable of playing multi-room audio you can set the device to Client Only mode.`

Reply:
Done!